app.rb

```diff
@@ -92,14 +93,9 @@
   redirect_flow_id = params[:redirect_flow_id]
   price = PACKAGE_PRICES.fetch(package)
 
-  # Complete the redirect flow
-  puts session[:token]
-  puts redirect_flow_id
```

Comment:
These weren't doing anything, probably left from debugging (when we were building the example app).